engines/misc/src/main/java/org/alfresco/transform/misc/transformers/TextToPdfContentTransformer.java

```diff
@@ -78,19 +91,29 @@ public class TextToPdfContentTransformer implements CustomTransformerFileAdaptor
     private static final byte BB = (byte) 0xBB;
     private static final byte BF = (byte) 0xBF;
 
+    @Value("${transform.core.misc.pdfBox.defaultFont}")

```

Comment:
This syntax usually fails if the property is not found, e.g.: a user overrides the property file and does not specify the `transform.core.misc.pdfBox.defaultFont` property. If that's the case, I think it could be sensible to provide a default on the code-level via the appropriate `@Value(property:default)` syntax as well, similarly to what we have done here: https://github.com/Alfresco/alfresco-transform-core/pull/885/files#diff-7b94454e8766efe02993468560e80b26b41e4dfeedb2361efef81b907cbe44beR29?

Reply:
I have added the default value at code level as suggested so we have an additional safeguard.